### DISCUSSION TITLE
Menyknapp som tab

### DISCUSSION THREAD

**Discussion by Murstam** [fremtind/jokul#1947] [Annet]
Hva tenker folk om å bruke menyknapp som tab slik som eksempelet her? Funker det eller må vi lage en tab-komponent?
<img width="1440" alt="Tilganger – Oversikt" src="https://user-images.githubusercontent.com/47025023/114039574-268f5f00-9883-11eb-8727-0e04a87c34f9.png">

**Comment by lfbergee** [comment]
Vi kan gjøre det, det er ikke krise. Men det er heller ikke optimalt, hvis vi ønsker at også skjermlesere for en best mulig opplevelse. Vil nok anbefale å lage en tab-komponent for dette. MVP utgaven kan være å implementere dette som to ulike sider og la tabbene være lenker.

Ref:
https://w3c.github.io/aria-practices/#tabpanel
https://www.deque.com/blog/a11y-support-series-part-1-aria-tab-panel-accessibility/

Eksempel:
https://pauljadam.com/demos/aria-tabpanel.html
